### PR TITLE
ncm-metaconfig: nginx add DHE-RSA-CHACHA20-POLY1305 cipher suite

### PR DESCRIPTION
**Pull request by wdpypere**

### CODE REVIEW

**Review by jrha — DISMISSED** [review]

**Comment by jrha** [comment]
This change would probably be more readable if there was one choice per line, but nevermind! :sweat_smile:

**Comment by wdpypere** [comment]
@jrha fixed

File: ncm-metaconfig/src/main/metaconfig/nginx/pan/schema.pan [hunk 1]
```diff
@@ -9,15 +9,51 @@ include 'pan/types';
...
+    "ECDHE-RSA-AES128-GCM-SHA256",
+    "ECDHE-RSA-AES128-SHA",
+    "ECDHE-RSA-AES128-SHA256",
+    "ECDHE-RSA-AES256-GCM-SHA384",
+    "ECDHE-RSA-AES256-SHA",
+    "ECDHE-RSA-AES256-SHA384",
+    "ECDHE-RSA-CHACHA20-POLY1305",
+    "ECDHE-RSA-DES-CBC3-SHA",
+    "EDH-RSA-DES-CBC3-SHA",
+    "TLSv1",
```
Comment (jrha) [inline ncm-metaconfig/src/main/metaconfig/nginx/pan/schema.pan:55]: Ooh, thanks! Looks like test fail due to `choice` not being able to handle a trailing comma here :disappointed:

Comment (wdpypere) [reply ncm-metaconfig/src/main/metaconfig/nginx/pan/schema.pan]: done

**Review by jrha — APPROVED** [review]

**Comment by jrha** [comment]
Thanks, I've also opened https://github.com/quattor/pan/issues/253

**Pull request merged**